media/src/App.svelte

```diff
@@ -475,11 +475,14 @@
             }
         } else if (data.message.pollResponse) {
             const sessionKey = data.message.session_key;
-            pollResponses[sessionKey] = data.message.pollResponse;
```

Comment:
This will need to be adjusted a bit to maintain functionality with the more basic poll types - the multiple choice response needs this line present.

Reply:
Since my adjustments have to check for the pollType anyways I just added an else condition.